eng/pipelines/spec-gen-sdk.yml

```diff
@@ -19,9 +19,18 @@ parameters:
   - name: SkipPullRequestCreation
     type: boolean
     default: false
-    
-trigger: none
-pr: none
+
+trigger:
+  branches:
+    include:
+      - user/raych1/support-pr-trigger
```

Comment:
This trigger is for testing purpose and will be deleted before rolling out pipeline.